### PR TITLE
Validation results tab

### PR DESCRIPTION
**Pull request by ebrockainq**

### CODE REVIEW

File: src/components/direct/hisp/ValidationResultsTab.tsx [hunk 1]
```diff
@@ -1,5 +1,52 @@
+import * as React from 'react'
+import { Box, Grid, Typography } from '@mui/material'
+import ProfilesCard from './ProfilesCard'
+
 const ValidationResults = () => {
-  return 'ValidationResults Tab'
+  return (
+    <Box
```
Comment (mattystank) [inline src/components/direct/hisp/ValidationResultsTab.tsx:7]: There should be a MUI Container wrapping the elements. Please reference other pages that have been built.

File: src/components/direct/hisp/ValidationResultsTab.tsx [hunk 2]
```diff
@@ -1,5 +1,52 @@
...
+        width: '100%',
+        minHeight: 'auto',
+        pl: '200px',
+      }}
+    >
+      <Typography variant="h4" sx={{ mb: 4 }}>
+        Below are the different profiles you are aligned to. Select one to see the validation results.
+      </Typography>
+
+      <Grid container spacing={4} sx={{ justifyContent: 'flex-start' }}>
```
Comment (mattystank) [inline src/components/direct/hisp/ValidationResultsTab.tsx:19]: I dont mind you using grid here.. but it needs to be responsive/scale.  Grid can be challenged. I suggest using box method like other pages

File: src/components/direct/hisp/ValidationResultsTab.tsx [hunk 3]
```diff
@@ -1,5 +1,52 @@
...
+        <Grid item sx={{ minWidth: 419 }}>
+          <ProfilesCard header=".gov" smtpAddress="direct.ett.health.gov" emailAddress="direct.ett.health.gov" />
+        </Grid>
+        <Grid item sx={{ minWidth: 419 }}>
```
Comment (mattystank) [inline src/components/direct/hisp/ValidationResultsTab.tsx:23]: This min width will need to change once you have a container.

File: src/components/direct/hisp/ValidationResultsTab.tsx (hunk 2 above)
Comment (ebrockainq) [reply src/components/direct/hisp/ValidationResultsTab.tsx:19]: Yeah I was trying something new, I'll switch it back to a box

File: src/components/direct/hisp/ProfilesCard.tsx [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+        height: 225,
+        boxShadow: '-8px 0px 32px 0px rgba(0, 0, 0, 0.17)',
+        mb: 4,
+      }}
+    >
+      <Toolbar>
+        <Typography variant="h4" sx={{ flexGrow: 1, fontWeight: 'bold', color: '#001439' }}>
+          {header}
+        </Typography>
+        <IconButton color="inherit">
```
Comment (mattystank) [inline src/components/direct/hisp/ProfilesCard.tsx:24]: Change this to Avatar component

File: src/components/direct/hisp/ProfilesCard.tsx [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+            <Typography variant="subtitle1">SUTE SMTP Address:</Typography>
+            <Typography variant="body1">{smtpAddress}</Typography>
+          </Grid>
+          <Grid item xs={12} sm={6}>
+            <Typography variant="subtitle1">SUTE Email Address:</Typography>
+            <Typography variant="body1">{emailAddress}</Typography>
+          </Grid>
+        </Grid>
+        <Divider sx={{ marginY: 2 }} />
+        <Button variant="contained" color="primary">
```
Comment (mattystank) [inline src/components/direct/hisp/ProfilesCard.tsx:41]: variant for button should be "outlined"

File: src/components/direct/hisp/ValidationResultsTab.tsx [hunk 6]
```diff
@@ -1,5 +1,43 @@
...
+import ProfilesCard from './ProfilesCard'
+
 const ValidationResults = () => {
-  return 'ValidationResults Tab'
+  return (
+    <Container>
+      <Box
+        sx={{
+          padding: '32px',
+          width: '1391px',
```
Comment (mattystank) [inline src/components/direct/hisp/ValidationResultsTab.tsx:11]: You shouldnt need this width

File: src/components/direct/hisp/ProfilesCard.tsx [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+            <Typography variant="body1">{emailAddress}</Typography>
+          </Grid>
+        </Grid>
+        <Divider sx={{ marginY: 2 }} />
+        <Button
+          variant="outlined"
+          color="primary"
+          sx={{
+            boxShadow:
+              '0px 1px 5px 0px rgba(0, 0, 0, 0.12), 0px 2px 2px 0px rgba(0, 0, 0, 0.14), 0px 3px 1px -2px rgba(0, 0, 0, 0.2)',
```
Comment (mattystank) [inline src/components/direct/hisp/ProfilesCard.tsx:43]: take out box shadow, please

**Pull request merged**